packages/cli/package.json

```diff
@@ -127,7 +127,7 @@
     "preact": "^10.5.13",
     "preact-render-to-string": "^5.1.19",
     "preact-router": "^3.0.1",
-    "puppeteer": "^9.1.1",
+    "puppeteer": "^13.7.0",
```

Comment:
This shouldn't be necessary, but our version of Puppeteer has turned *super* flakey recently. Granted it probably is a good few years old, so maybe that's unavoidable.